custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -292,18 +297,27 @@ def _followups(self):
 
     @property
     def state(self):
+        if self.test_phi is not None:
+            return MockLocation('FAKESTATE', 'fake_state_id', MockLocationType('state', 'state'))
```

Comment:
so far I've gotten by with just setting a few nikshay codes manually to test Locations, but if that becomes a blocker I'll take this suggestion